etc/kayobe/kolla/config/nova/nova-compute-ironic.conf

```diff
@@ -2,3 +2,16 @@
 [DEFAULT]
```

Comment:
Hey, that should probably be outside of the `{% if %}` now that there is other stuff under it.